scipy/signal/_fir_filter_design.py

```diff
@@ -1302,79 +1302,41 @@ def fwind1(hsize, window, *, fc=None, fs=2, circular=False):
         Desired window to use for each 1D filter or a single window type 
         for creating circularly symmetric 2-D windows. Each element should be
         a string or tuple of string and parameter values. See
-        '~scipy.signal.get_window' for a list of windows and required
+        `~scipy.signal.get_window` for a list of windows and required
         parameters.
     fc : float or 1-D array_like, optional
-        Cutoff frequency of filter (expressed in the same units as `fs`).
-        Required if `circular` is False.
+        Cutoff frequency of the filter in the same units as `fs`. This defines
+        the frequency at which the filter's gain drops to approximately -6 dB
+        (half power) in a low-pass or high-pass filter. For multi-band filters,
+        `fc` can be an array of cutoff frequencies (i.e., band edges) in the 
+        range [0, fs/2], with each band specified in pairs. Required if 
+        `circular` is False.
     fs : float, optional
         The sampling frequency of the signal. Default is 2.
     circular : bool, optional
         Whether to create a circularly symmetric 2-D window. Default is False.
+    pass_zero : {True, False, 'bandpass', 'lowpass', 'highpass', 'bandstop'}, optional
+        If True, the gain at the frequency 0 (i.e., the "DC gain") is 1.
+        If False, the DC gain is 0. Can also be a string argument for the
+        desired filter type (equivalent to ``btype`` in IIR design functions).
```

Comment:
We are in 2D now—better would be:
```text
This parameter is passed to the `firwin` function for each scalar frequency axis.
Hence, if ``True``, the DC gain, i.e., the gain at frequency (0, 0), is 1.
If ``False``, the DC gain is 0 at frequency (0, 0) if `circular` is ``True``. 
If `circular` is ``False`` the frequencies (0, f1) and (f0, 0) will have gain 0.
It can also be a string argument for the desired filter type 
(equivalent to ``btype`` in IIR design functions).
```